src/authInfo.ts

```diff
@@ -1117,16 +1120,17 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
     // within this file to support it.
     const apiVersion = 'v51.0'; // hardcoding to v51.0 just for this call is okay.
     const instance = ensure(instanceUrl);
-    const userInfoUrl = `${instance}/services/oauth2/userinfo`;
+    const baseUrl = new URL(instance);
+    const userInfoUrl = `${baseUrl}services/oauth2/userinfo`;
```

Comment:
is there a slash or not (vs. line 1133)

Reply:
@mshanemc since the function was using the value sent from the user it could end up with a double slash, which is why I passed it through new URL, which guarantees the URL will have a slash on the end.